sygnal/notifications.py

```diff
@@ -63,24 +64,18 @@ def __init__(self, raw):
 
 class Notification:
     def __init__(self, notif):
-        optional_attrs = [
-            "room_name",
-            "room_alias",
-            "prio",
-            "membership",
-            "sender_display_name",
-            "content",
-            "event_id",
-            "room_id",
-            "user_is_target",
-            "type",
-            "sender",
-        ]
-        for a in optional_attrs:
-            if a in notif:
-                self.__dict__[a] = notif[a]
-            else:
-                self.__dict__[a] = None
+        # optional attributes
+        self.room_name: Optional[str] = notif.get("room_name")
+        self.room_alias: Optional[str] = notif.get("room_alias")
+        self.prio: Optional[str] = notif.get("prio")
+        self.membership: Optional[str] = notif.get("membership")
+        self.sender_display_name: Optional[str] = notif.get("sender_display_name")
+        self.content = notif.get("content")
```

Comment:
I think you missed this one?

```suggestion
        self.content: Optional[str] = notif.get("content")
```

Reply:
I did miss that one, but the type of content appears to be a dict (I should check the spec to be sure that there is no other option on top of that, though).